### PR TITLE
🚸 Clarify language for required files (with references to examples)

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/186#issuecomment-2269154543 "IC_kwDOKRPTI86HQIzv")

Clarify language for required files (this ties into Policy Panda, too)

voting group: @commonhaus/cf-egc 

Do one of the following:

- Approve the PR or react with 👍 (:+1:) if it looks good to you
- Review with Comments or react with 👀 (:eyes:) if you're "ok" with it (it may not be your favorite)
- If you think it needs discussion or revision
    - Create a review, add your comments and require changes
    - Use the +- button to make a suggestion (instead of just adding a comment).

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
-    - Explanation of the difference between DCO and CLA: <https://www.consortiuminfo.org/open-source/all-about-clas-and-dcos/>
+- If your project uses a CLA, include that requirement in the contribution guide. [For example](https://github.com/jreleaser/jreleaser/blob/main/CONTRIBUTING.adoc#contributor-license-agreement):
 
-- **CODEOWNERS** to define individuals or teams that are responsible for code in a repository
-    - [About Code Owners (GitHub)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
-    - [Code Owners (GitLab)](https://docs.gitlab.com/ee/user/project/codeowners/)
+    ```md
+    Contributor License Agreement
+
+    Contributions to JReleaser are protected by a CLA.
```
Comment (henri-tremblay) [inline templates/README.md:45]: Is it intended to mention JRelease here as an example or it should be MY_PROJECT or something like this?

**Review by henri-tremblay — APPROVED** [review]

File: templates/README.md (hunk 1 above)
Comment (ebullient) [reply templates/README.md:45]: This is an example. Happy for you to suggest the change to make it more readable / more obviously an example.

Comment (henri-tremblay) [reply templates/README.md:45]: `Contributions to My_Cool_Commonhaus_Project`?

Comment (ebullient) [reply templates/README.md:45]: Sure. I will train you yet.. use the +- thing so I can just commit your suggestion. ;)

Comment (henri-tremblay) [inline templates/README.md:45]: Contributions to My_Cool_Commonhaus_Project are protected by a CLA.

Comment (henri-tremblay) [reply templates/README.md:45]:
```suggestion
    Contributions to JReleaser are protected by a CLA.
```

**Review by cealsair — APPROVED** [review]

File: templates/README.md [hunk 2]
```diff
@@ -1,41 +1,52 @@
...
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Governance](GOVERNANCE.md)
+[^1]: This file can also be defined in the `.github` repository of an organization.
 
-## Other files
+We further recommend:
+
+- **Code Ownership:** Maintain a list of individuals or groups with the authority to review and merge contributions (`CODEOWNERS` or `.github/CODEOWNERS`).
+    - [About Code Owners (GitHub)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+    - [Code Owners (GitLab)](https://docs.gitlab.com/ee/user/project/codeowners/)
```
Comment (gavinking) [inline templates/README.md:15]: I would remove this.

If a project thinks a `CODEOWNERS` file is useful, nothing stops the project having it. CF doesn't need to "recommend" it.

```suggestion
```

Comment (ebullient) [reply templates/README.md:15]: You may not use it with Hibernate, but it remains something the foundation would recommend (as it is conventional, supported by tools, and can make required reviewers more transparent). It is a recommendation, not a requirement, and it is not made idly.

Comment (gavinking) [reply templates/README.md:15]: How many of the Foundation's current projects use this?

Comment (aalmiray) [reply templates/README.md:15]: JReleaser for one set up CODEOWNERS long time ago, I think to follow the recommendations of the OSSF Scorecard. The project will follow all if nos most of the recommendations laid out by CF once this PR is merged.

**Review by evanchooly — APPROVED** [review]

File: templates/README.md [hunk 3]
```diff
@@ -1,41 +1,52 @@
 # Commonhaus Templates
 
-We don't want project on-boarding to be a burden.
+We don't want project on-boarding to be a burden, but we have a few requirements to ensure transparency.
 
-The files in this folder are our recommendations for what CF projects should define.
+- `CODE_OF_CONDUCT.md`[^1]: Projects must define a Code of Conduct. They can use the foundation's [Code of Conduct policy][coc_policy], or define their own. See [CODE_OF_CONDUCT.md][COC] and the [Project CODE_OF_CONDUCT.md template][COC_TPL] for examples.
```
Comment (maxandersen) [inline templates/README.md:5]: panda check allows .md,.adoc,.rst,.txt ...should i make it only greenlight .md ?

Comment (ebullient) [reply templates/README.md:5]: I'm ok with addition that other txt-based formats are ok, too. Has to be obvious name, has to be plain text. Markup grammar is negotiable.

**Comment by ebullient** [comment]
vote::result
- after discussion with @jkschneider, we can/should proceed on some items without waiting for @timtebeek
- otherwise, no disagreement

**Pull request merged**